cloudformation/template.yml

```diff
@@ -160,6 +161,10 @@ Parameters:
       A list of IDs of subnets in which to launch Batch instances (all subnets
       must exist in the VPC you selected)
 
+Conditions:
+  UseHostedDockerImage: !Equals [!Ref RepositoryName, ""]
+  UseCustomDockerImage: !Not [!Equals [!Ref RepositoryName, ""]]
```

Comment:
We might be able to get away with avoiding this explicit condition and instead using `!Not [UseHostedDockerImage]`, but I figured being a little bit more verbose is useful here anyway.